velox/core/QueryConfig.h

```diff
@@ -567,6 +575,10 @@ class QueryConfig {
     return get<bool>(kValidateOutputFromOperators, false);
   }
 
+  bool enableExpressionEvaluationCache() const {
```

Comment:
isExpressionEvaluationCacheEnabled to avoid giving an impression that this method enables the cache